x-pack/plugins/infra/public/pages/metrics/hosts_landing/index.tsx

```diff
@@ -0,0 +1,90 @@
+ * 2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { EuiButton, EuiCallOut } from '@elastic/eui';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { enableInfrastructureHostsView } from '@kbn/observability-plugin/public';
+import { HostsPage } from '../hosts';
+import { EnableHostViewPage } from './enable_host_view_page';
+
+export const HostsLandingPage = () => {
+  const kibana = useKibana();
+  const canEditAdvancedSettings = kibana.services.application?.capabilities.advancedSettings.save;
+  const [isHostViewEnabled, setIsHostViewEnabled] = useState<boolean | undefined>(undefined);
+
+  const ROLE = i18n.translate('xpack.infra.hostsLandingPage.role', {
+    defaultMessage: 'role',
+  });
+
+  useEffect(() => {
+    setIsHostViewEnabled(kibana.services.uiSettings?.get(enableInfrastructureHostsView));
+  }, [isHostViewEnabled, kibana.services.uiSettings]);
+
+  if (isHostViewEnabled) {
+    return <HostsPage />;
+  }
+
+  if (canEditAdvancedSettings) {
+    return (
+      <EnableHostViewPage
+        actions={
+          <EuiButton
+            iconType="check"
+            color="primary"
+            data-test-subj="hostsView-enable-feature-button"
+            onClick={() => {
+              kibana.services.uiSettings?.set(enableInfrastructureHostsView, true);
+              setIsHostViewEnabled(true);
+            }}
+          >
+            {i18n.translate('xpack.infra.hostsLandingPage.enableHostsView', {
+              defaultMessage: 'Enable hosts view',
+            })}
+          </EuiButton>
+        }
+      />
+    );
+  }
+
+  return (
+    <EnableHostViewPage
+      actions={
+        <EuiCallOut data-test-subj="hostView-no-enable-access" size="s" color="warning">
+          <p>
+            {i18n.translate(
+              'xpack.infra.hostsLandingPage.calloutReachOutToYourKibanaAdministrator',
+              {
+                defaultMessage: `Your user role doesn’t have sufficient privileges to enable this feature - please 
+                reach out to your Kibana Administrator and ask them to visit this page to enable this feature.`,
+              }
+            )}
+          </p>
+          <p>
+            <FormattedMessage
+              id="xpack.infra.hostsLandingPage.calloutRoleClarificationWithDocsLink"
+              defaultMessage="They will need a {docsLink} with Kibana > Management > Advanced Settings > All permissions."
+              values={{
+                docsLink: (
+                  <EuiLink
+                    data-test-subj="hostsView-role-docs-link"
+                    href="https://www.elastic.co/guide/en/kibana/current/kibana-privileges.html#kibana-feature-privileges"
+                  >
+                    {ROLE}
+                  </EuiLink>
```

Comment:
```suggestion
                  <EuiLink
                    data-test-subj="hostsView-role-docs-link"
                    href="https://www.elastic.co/guide/en/kibana/current/kibana-privileges.html#kibana-feature-privileges" 
                    external
                  >
                    {ROLE}
                  </EuiLink>
```

Reply:
just adding the `external` prop to the link so it opens in a new window and indicates that it's opening in a new window.

Reply:
I added `target="_blank"` which will default `external` to `true` and will also open the link in a new tab ( with the `external` prop only the indicator icon was there but the link still opened in the same tab )

Reply:
@jennypavlova Ah, thanks for double-checking that 👍